Examples/Swift/DraggableAnnotationViewExample.swift

```diff
@@ -115,6 +115,12 @@ class DraggableAnnotationView: MGLAnnotationView {
             self.layer.opacity = 0.8
             self.transform = CGAffineTransform.identity.scaledBy(x: 1.5, y: 1.5)
         }, completion: nil)
+
+        // Initialize haptic feedback generator and give the user a light thud.
+        if #available(iOS 10.0, *) {
+            let hapticFeedback = UIImpactFeedbackGenerator(style: .light)
```

Comment:
Should we add a note about not being able to store the potentially unavailable object as a property?

Reply:
I dunno — it's a system limitation that is annoying but isn’t ultimately a huge deal, so we don’t strictly have to mention it. It verges on being a bit too meta, I’d say.

If anyone feels strongly about it and can think of a good way to say, “Typically you’d reuse the feedback generator but we can’t because Swift and backwards compatibility” — please feel free. 🤔